pages/sponsor/index.vue

```diff
@@ -7,7 +7,8 @@
                 :key="`sponsor_intro_${i}`"
                 :path="`intro.${i}`"
                 tag="p"
-                class="text-base"
+                class="text-xs md:text-sm"
+                style="line-height: 176%"
```

Comment:
Based on the current convention, please avoid inline style change. Alternatively, you can have a dedicated class and configure the css for the class in the `style` block.

Reply:
ok